Corona Tracker/index.html

```diff
@@ -91,11 +91,52 @@ <h6 id = "country-deaths">Select a Country</h6>
+                <li>Stay home and self-isolate even with minor symptoms such as cough, headache, mild fever, until you recover. Have someone bring you supplies. If you need to leave your house, wear a mask to avoid infecting others. Why? Avoiding contact with others will protect them from possible COVID-19 and other viruses.</li>
+                <li>If you have a fever, cough and difficulty breathing, seek medical attention, but call by telephone in advance if possible and follow the directions of your local health authority. Why? National and local authorities will have the most up to date information on the situation in your area. Calling in advance will allow your health care provider to quickly direct you to the right health facility. This will also protect you and help prevent spread of viruses and other infections.</li>
+                <li>Keep up to date on the latest information from trusted sources, such as WHO or your local and national health authorities. Why? Local and national authorities are best placed to advise on what people in your area should be doing to protect themselves.</li>
+            </ul>
+        
+        </details>
+        <details>
+            <summary>
+                Advice on the safe use of alcohol-based hand sanitizers
+            </summary>
+            <p>
+                To protect yourself and others against COVID-19, clean your hands frequently and thoroughly. Use alcohol-based hand sanitizer or wash your hands with soap and water. If you use an alcohol-based hand sanitizer, make sure you use and store it carefully.
+            </p>
+            <ul>
+                <li> Keep alcohol-based hand sanitizers out of children’s reach. Teach them how to apply the sanitizer and monitor its use.</li>
+                <li> Apply a coin-sized amount on your hands. There is no need to use a large amount of the product.</li>
+                <li> Avoid touching your eyes, mouth and nose immediately after using an alcohol-based hand sanitizer, as it can cause irritation.</li>
+                <li>Hand sanitizers recommended to protect against COVID-19 are alcohol-based and therefore can be flammable. Do not use before handling fire or cooking. </li>
+                <li> Under no circumstance, drink or let children swallow an alcohol-based hand sanitizer. It can be poisonous. </li>
+                <li> Remember that washing your hands with soap and water is also effective against COVID-19. </li>
+            </ul>
+        
+        </details>
+    </section>
+
+
     <div class = "container">
-        <label for="selection">Made with love by theroyakash 2020</label>
+        <label for="selection">Made with love by theroyakash & amder111 2020</label>

```

Comment:
If you could add the heref links to theroyakash with www.iamroyakash.com and amder111 to your linkedin link

Reply:
Okay I’ll do it asap

Reply:
Appreciate it